recipe/build.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
```

Comment:
Is there a libedit in the $PREFIX/lib? Is there something extra we need for this/